### PR TITLE
Update Stat Var Explorer

### PR DESCRIPTION
**Pull request by juliawu**
Makes the following updates to the stat var explorer:
* Changes the landing page of the explorer to have instructions matching @miss-o-soup's mocks.
* Updates the styling of the variable sidebar at the left
* Fixes a bug where the stat vars are not clickable when the sidebar is modal

As part of these changes, also makes the following more global changes:
* Updates the default body font to "Google Sans" so the fonts used on stat var and data download tool can match fonts used on rest of the website
* Reduces the padding on the footer for all tools so that there's more room for visualizations
* Removes the subpage title on each page, this space will be given to vertical titles like "Biomedical" in the future.

Also tested changes on custom DC. Changes there match the look of main DC.

### Desktop
localhost_8080_tools_statvar

localhost_8080_tools_statvar (2)

### Moblie
localhost_8080_tools_statvar(iPad Mini) (2)

localhost_8080_tools_statvar(iPad Mini) (1)

## Other Tools

localhost_8080_tools_download

localhost_8080_tools_timeline

localhost_8080_tools_visualization

### CODE REVIEW

File: static/css/stat_var_hierarchy.scss
Comment (chejennifer) [inline static/css/stat_var_hierarchy.scss:28]: remove the old font-size and line-height?

File: static/css/tools/base_tools.scss
Comment (chejennifer) [inline static/css/tools/base_tools.scss:48]: I don't think this will be needed anymore when my PR goes in

File: static/css/tools/stat_var.scss
Comment (chejennifer) [inline static/css/tools/stat_var.scss:124]: hm, this looks like a few too many characters for a color hex .. can you double check this

File: static/js/tools/stat_var/info.tsx
Comment (chejennifer) [inline static/js/tools/stat_var/info.tsx:39]: curious what this empty section is for?

**Review by chejennifer — COMMENTED** [review]
nice!

File: server/templates/base.html
Comment (miss-o-soup) [reply server/templates/base.html:93]: @chejennifer - What is the subpage that you are referring too, maybe yes, but want to see before I vote ;-)

Comment (chejennifer) [reply server/templates/base.html:93]: this piece of code refers to this part of the header: https://screenshot.googleplex.com/9556CVikX5ZHQat

& usually clicking it will take the user back to the landing page for that tool

Comment (juliawu) [reply server/templates/base.html:93]: Thanks, also removed subpage_title from other pages on main DC and biomedical (but left in custom dc, which uses custom dc base template)

Change is already in description, I had called it subtitle instead of subpage_title before.

File: server/webdriver/tests/ranking_test.py
Comment (juliawu) [reply server/webdriver/tests/ranking_test.py:65]: My mistake, only staged a halfway change and forgot to stage the other half before comiting. This line is now just tweaked to reflect the removal of subpage_titles.

File: static/css/stat_var_hierarchy.scss
Comment (juliawu) [reply static/css/stat_var_hierarchy.scss:28]: Good catch, removed the old one.

File: static/css/tools/stat_var.scss
Comment (juliawu) [reply static/css/tools/stat_var.scss:124]: This is a hexadecimal color with transparency: #c2e7ff with transparency 33 (20% opacity).

[See https://www.w3schools.com/cssref/tryit.php?filename=trycss_color_hexa](https://www.w3schools.com/cssref/css_colors_legal.php) for more info.

File: static/js/tools/stat_var/info.tsx
Comment (juliawu) [reply static/js/tools/stat_var/info.tsx:39]: Good catch, removed.

File: static/css/tools/base_tools.scss
Comment (juliawu) [reply static/css/tools/base_tools.scss]: Ack, this is now reflected after merging with current master.

File: static/css/stat_var_hierarchy.scss
Comment (chejennifer) [reply static/css/stat_var_hierarchy.scss:27]: there are still 2 font-sizes?

**Review by chejennifer — APPROVED** [review]
thanks for the changes! lgtm but lets wait on the decision about subpage_title

File: static/css/stat_var_hierarchy.scss
Comment (juliawu) [reply static/css/stat_var_hierarchy.scss:26]: Fixed!

**Comment by juliawu** [comment]
Per conversation with @miss-o-soup, confirmed we are removing the subpage_titles globally. Merging.

**Pull request merged**
